inst/stan/estimate_infections.stan

```diff
@@ -96,18 +96,27 @@ transformed parameters {
   if (estimate_r) {
     // via Rt
     vector[gt_max[1]] gt_pmf;
-    gt_pmf = combine_pmfs(gt_fixed_pmf, gt_mean, gt_sd, gt_max, gt_dist, gt_max[1], 1);
-    R = update_Rt(ot_h, log_R[estimate_r], noise, breakpoints, bp_effects, stationary);
-    infections = generate_infections(R, seeding_time, gt_pmf, initial_infections, initial_growth, pop, future_time);
+    gt_pmf = combine_pmfs(
+      gt_fixed_pmf, gt_mean, gt_sd, gt_max, gt_dist, gt_max[1], 1, 1
+    );
+    R = update_Rt(
+      ot_h, log_R[estimate_r], noise, breakpoints, bp_effects, stationary
+    );
+    infections = generate_infections(
+      R, seeding_time, gt_pmf, initial_infections, initial_growth, pop,
+      future_time
+    );
   } else {
     // via deconvolution
-    infections = deconvolve_infections(shifted_cases, noise, fixed, backcalc_prior);
+    infections = deconvolve_infections(
+      shifted_cases, noise, fixed, backcalc_prior
+    );
```

Comment:
Can we separate style commits vs code commits? It takes some looking to see what exactly has changed here in functionality and it's easy to miss something.